### DISCUSSION TITLE
How to open the options window?

### DISCUSSION THREAD

**Discussion by trx32** [pubpub-zz/ppInk#325] [Q&A] [answered]
Hi, I love the app, but I can't seem to find the options screen for settings: like where to find the screen caps and cursor settings on open.
How do I open the options window? I downloaded what I think is the latest version but now sure how to find the exact release.
Thanks!

**Comment by pubpub-zz** [comment, marked as answer]
you will find the  options menus when you right click on the ppink icon in the notification area (different from the floating window)
the folder where the screenshot is avaible in the window.
some options are not available in the window but are only accessible in the config.ini file
for the cursors, you need to replace the existing files with your own

do not hesitate to star the project 😉

**Reply by trx32** [reply]
Thanks, That is very helpful!
